src/main/java/eu/eutampieri/catacombs/ui/Game.java

```diff
@@ -248,10 +248,10 @@ public final void run() {
             timer += now - lastTime;
             updates = 0;
             while ((now - lastUpdateTime) >= tickPerTime) {
-                float delta;
-                delta = (now - lastUpdateTime) / DELTA_CONST;
+                int delta;
+                delta = Math.round((now - lastUpdateTime) / DELTA_CONST);
                 KEY_MANAGER.update(delta);
-                delta = delta <= DELTA_MIN ? delta : DELTA_MIN;
+                delta = Math.round(delta <= DELTA_MIN ? delta : DELTA_MIN);
```

Comment:
Is this needed? Perhaps just make DELTA_MIN an `int`

Reply:
math.round might not be needed using integers as well